test/ts-parser.test.js

```diff
@@ -5,10 +5,12 @@ var expect = require('chai').expect;
 var TSParser = require('../lib/tsParser');
 
 describe('TypeScript Parser Test', function() {
-  var tsOptions = {tsconfig: path.join(__dirname, 'tsconfig.json')};
+  var tsOptions = {
+    excludeNotExported: false,
```

Comment:
Any reason this has to be set to false for tests? `fixtures/ts/Greeter*.ts` seem to be exporting the classes.

Reply:
@virkt25 Great catch! That led to https://github.com/TypeStrong/typedoc/pull/694.